tools/add_triage_signature.py

```diff
@@ -911,6 +911,44 @@ def _update_ticket(self, url, issue_key, should_update=False):
             self._update_triaging_ticket(issue_key, report, should_update=should_update)
 
 
+
+class CoreOSInstallerErrorSignature(ErrorSignature):
+    '''
+    The signature looks for coreos-installer errors, usually due to bad disks
+    '''
+
+    ERROR_PATTERN = r'coreos-installer install .* Error exit status'
```

Comment:
Won't it catch also media disconnection issues? Maybe worth adding this signature only when 'Unable to read from the discovery media' is not in the status?